packages/metadata/src/services/google.ts

```diff
@@ -224,20 +231,22 @@ class Client {
                         clientId: Client.clientId,
                         code,
                         codeVerifier: random,
-                        redirectUri,
+                        redirectUri: Client.receiverUrl,
                     }),
                     headers: {
                         'Content-Type': 'application/json',
                     },
                 });
 
                 const json = await res.json();
+
                 if (!json?.access_token || !json?.refresh_token) {
                     throw new Error('Could not retrieve the tokens.');
                 }
                 Client.accessToken = json.access_token;
                 Client.refreshToken = json.refresh_token;
             } catch {
+                // TODO: it looks like that this actually does not work. I tested it even on develop by throwing an artifical error somewhere above.
```

Comment:
cc @komret. not a big deal. if I am not wrong, this doesn't work. at least it helped me discover another potential runtime issue I  fixed here https://github.com/trezor/trezor-suite/pull/16110/files#diff-56d44c230c949c29acb06aed65bf1fffef52e38cf29e86546c87b0e92b2c9c25R165